cdisc_rules_engine/rules_engine.py

```diff
@@ -240,7 +256,18 @@ def validate_rule(
                     self.standard, self.standard_version
                 )
             )
-            define_metadata: List[dict] = builder.get_define_xml_variables_metadata()
+            if vx in ("Y", "YES"):
```

Comment:
I think we can isolate this check in the define_xml_reader class instead of checking it at this level. That way the engine doesn't care if the validate xml flag is set, it just runs rules

Reply:
Could we do it in another ticket for up-lifting it to reader level?

Reply:
I would rather not since it would just be knowingly adding tech debt, but if @gerrycampion agrees to do it later, I would be ok with creating a new issue for it

Reply:
didnt see this before - i agree with nic that we don't need this outside the define_xml_reader